### ISSUE TITLE
Add to Bower

### ISSUE THREAD

**Issue by pudgereyem** [resrcit/ReVIEW#2]
Make this a bower repository :)
Instructions can be found here; http://bower.io/docs/creating-packages/

**Comment by domfee** [comment]
@pudgereyem Yep its been on my todo list for some time. Same goes for: https://github.com/resrcit/resrc.js also

**Comment by pudgereyem** [comment]
Awesome. I'd love to try https://github.com/resrcit/resrc.js in my next project. I'm using https://responsive.io/ today (which works great I think). I haven't been using any advanced features, but what would you say is the difference between http://www.resrc.it/ and https://responsive.io/? Anything that sets you apart?

On topic: I actually went with https://github.com/luis-almeida/unveil just because it was listed in bower. Just a few lines of code ;) (https://github.com/luis-almeida/unveil/blob/master/bower.json)

**Comment by domfee** [comment]
@pudgereyem Yes definitely need to get bower added into the repo. I'll get this sorted. 

At first glance I think the main differences are:

* Offline image support for local development / intranets (http://www.resrc.it/blog/roxy-proxy-server)
* [JPEGmini](http://www.jpegmini.com/) integration (75% smaller images without compromising quality) on ALL plans.
* We have our OWN CDN technology, we don't relay on a 3rd party. I believe they use Cloudflare
* Total Control to layer your own CDN over the top
* Open-Source Javascript and Plugin Support
* Image Cache control
* Developer REST API
* Real-time Analytics

[More Features Listed Here](http://www.resrc.it/pricing)

**Comment by pudgereyem** [comment]
@domfee awesome, I'll try it out on my next project and let you know what I think.

**Comment by domfee** [comment]
@pudgereyem Thumbs Up!
